fetch/range/resources/long-wav.py

```diff
@@ -50,11 +50,28 @@ def main(request, response):
     response.headers.set("Cache-Control", "no-cache")
 
     range_header = request.headers.get('Range', '')
+    range_header_match = range_header and re.search(r'^bytes=(\d*)-(\d*)$', range_header)
     range_received_key = request.GET.first('range-received-key', '')
+    accept_encoding_key = request.GET.first('accept-encoding-key', '')
 
     if range_received_key and range_header:
+        # Remove any current value
+        request.server.stash.take(range_received_key, '/fetch/range/')
         # This is later collected using stash-take.py
-        request.stash.put(range_received_key, 'range-header-received', '/fetch/range/')
+        request.server.stash.put(range_received_key, 'range-header-received', '/fetch/range/')
+
+    if accept_encoding_key:
+        # Remove any current value
+        request.server.stash.take(
+            accept_encoding_key,
+            '/fetch/range/'
+        )
```

Comment:
Why is this needed?

Reply:
Safari appears to make two requests in quick succession. Without this, an error is thrown, since it tries to put something in the stash when that key already exists.